test/e2e/specs/editor/blocks/block-title.spec.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post Title block', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'Can edit the post title', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'core/post-title' } );
+
+		// Add the post title
+		await editor.canvas
+			.getByRole( 'textbox', {
+				name: 'Add title',
+			} )
+			.fill( 'Just tweaking the post title' );
+
+		// Save the post draft and reload.
+		await page.getByRole( 'button', { name: 'Save draft' } ).click();
+		await expect(
+			page
+				.getByRole( 'button', { name: 'Dismiss this notice' } )
+				.filter( { hasText: 'Draft saved' } )
+		).toBeVisible();
```

Comment:
Maybe it's time to add the `editor.saveDraft()` helper. There are at least 20 occurrences of similar logic in the tests.

cc @WunderBart, @kevin940726

Reply:
Here's the PR #55308.